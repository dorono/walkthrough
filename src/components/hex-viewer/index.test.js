import React from 'react';
import {shallow} from 'enzyme';
import HexViewer from './index';

const defaultProps = {
    data: '',
};

describe('HexViewer', () => {
    it('should render without errors', () => {
        shallow(<HexViewer {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<HexViewer {...defaultProps} />)).toMatchSnapshot();
    });
});
