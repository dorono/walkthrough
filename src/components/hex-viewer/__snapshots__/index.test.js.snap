// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HexViewer should match snapshot 1`] = `
<pre
  className="root"
>
  <code />
</pre>
`;
